pandas/tests/groupby/test_timegrouper.py

```diff
@@ -571,8 +571,7 @@ def test_groupby_with_timezone_selection(self):
     def test_timezone_info(self):
         # GH 11682
         # Timezone info lost when broadcasting scalar datetime to DataFrame
-        tm._skip_if_no_pytz()
-        import pytz
+        pytz = pytest.importorskip("pytz")
```

Comment:
actually we should remove this entirely
we require pytz (prob lots of skips related to pytz in the codebase - should remove all of them)